.github/workflows/starknet-go-tests.yml

```diff
@@ -0,0 +1,33 @@
+name: starknet-go tests
+
+on:
+  workflow_call:
+    secrets:
+      TEST_RPC_URL:
+        required: false
```

Comment:
what happens when this is not provided?  I'm leaning toward thinking that these two are required...

Reply:
right, it should be rather required - updated